app/views/maily/emails/show.html.erb

```diff
@@ -18,17 +18,19 @@
     <% end %>
   </li>
 
-  <li class="splitter">|</li>
+  <% if Maily.allow_delivery %>
```

Comment:
Thanks for this PR @ArturMinnullin! 👌🏼 

It would be nice to change this line to `Maily.allowed_action?(:deliver)`, since the `.allowed_action?` method is like the public internal API to check for "permissions". Do you mind to change it?

Reply:
Yes, sure. I will change it today :)